crates/rpc/src/error.rs

```diff
@@ -18,17 +18,17 @@ pub enum RunError {
 }
 
 #[derive(Error, Debug)]
-pub enum BeerusRpcError {
+pub enum RpcError {
```

Comment:
I'm all for it because I find this kind of prefix redundant, however this is a breaking change.

I wonder if we could add something like this?
```rust
#[deprecated]
use self::RpcError as BeerusRpcError;
```